ReactWindows/ReactNative.Shared/UIManager/BorderedViewParentManager.cs

```diff
@@ -77,9 +86,13 @@ public void SetBorderRadius(Border view, int index, double radius)
             ViewProps.BackgroundColor,
             CustomType = "Color",
             DefaultUInt32 = ColorHelpers.Transparent)]
-        public void SetBackgroundColor(Border view, uint color)
+        public void SetBackgroundColor(TFrameworkElement view, uint color)
         {
-            view.Background = new SolidColorBrush(ColorHelpers.Parse(color));
+            if (!HasBorder(view) && ColorHelpers.IsTransparent(color))
+                return;
+
```

Comment:
I think at some point we relied on transparent borders because without them the views weren't picked up as click targets. However, we've since done a bunch of work with the TouchHandler, so it may no longer be a problem, can you confirm if you have buttons with transparent backgrounds that they can still be targeted with and w/o a border